ompi/group/group.h

```diff
@@ -379,19 +385,19 @@ static inline struct ompi_proc_t *ompi_group_dense_lookup (ompi_group_t *group,
  * This is the function that iterates through the sparse groups to the dense group
  * to reach the process pointer
  */
-static inline ompi_proc_t *ompi_group_get_proc_ptr (ompi_group_t *group, int rank, const bool allocate)
+static inline ompi_proc_t *ompi_group_get_proc_ptr (ompi_group_t *group, int rank, unsigned char flags)
```

Comment:
And here - wonder if the const can still be kept.

Reply:
I don't think it matters, these arguments are const by default as they cannot be returned to the caller.